docs/basic-usage.ipynb

```diff
@@ -111,18 +137,25 @@
   {
    "cell_type": "markdown",
    "id": "0cc4b64d",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "\n",
     "We can now visualise the number of people living in each cell (the cells are represented by their centre point):\n"
-    "\n",
```

Comment:
It seems that this is a known issue with Pycharm, i.e it can corrupt the Notebooks if you modify the file using Pycharm.. 🤦🏻‍♂️ https://youtrack.jetbrains.com/issue/DS-3228/Remove-newline-at-the-end-of-ipynb-file-and-stem-cell-metadata-i